.github/workflows/push.yml

```diff
@@ -0,0 +1,48 @@
+name: Node CI (push)
+on:
+  push:
+    branches:
+      - develop
+      - gh-pages
+
+jobs:
+  test-headless:
+    name: Headless Tests
+    runs-on: ubuntu-latest
+    needs: lint
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Cache node modules
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-12-${{ hashFiles('**/package-lock.json') }}
+      - name: install & build
+        run: |
+          npm ci
+          npm run build:w3c
+      - run: npm run test:headless
+
+  test-karma:
+    name: Karma Unit Tests (Chrome)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Cache node modules
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-12-${{ hashFiles('**/package-lock.json') }}
+      - name: install & build
+        run: |
+          npm ci
+          npm run build:w3c & npm run build:geonovum
```

Comment:
do we need to build:dini also?

Reply:
There are no tests for dini profile yet.

Reply:
True...